src/index/IndexMetaDataImpl.h

```diff
@@ -6,6 +6,7 @@
 #pragma once
 
 #include "../util/File.h"
+#include "../util/Serializer/ByteBufferSerializer.h"
 #include "../util/Serializer/FileSerializer.h"
 #include "../util/Serializer/SerializeHashMap.h"
```

Comment:
These can and should also all be `"util/Serializer/..."` (absolute paths) while you are at it.